.github/dependabot.yml

```diff
@@ -4,24 +4,54 @@ updates:
 - package-ecosystem: gradle
   directory: "/data-prepper-api"
   schedule:
-    interval: daily
-  open-pull-requests-limit: 3
+    interval: weekly
 
 - package-ecosystem: gradle
-  directory: "/data-prepper-benchmarks"
+  directory: "/data-prepper-core"
   schedule:
-    interval: daily
-  open-pull-requests-limit: 3
```

Comment:
Just for my information: what is the default limit ?

Reply:
5 is the default